src/iris/utils.py

```diff
@@ -9,17 +9,29 @@
 from gevent import sleep
 import datetime
 import ujson
+import os
 from . import db
+from iris.config import load_config
 import re
 import msgpack
 import logging
+from random import randrange
 
 logger = logging.getLogger(__name__)
 
+config = {}
+if 'IRISPYTEST' not in os.environ:
+    config = load_config()
+
 uuid4hex = re.compile('[0-9a-f]{32}\Z', re.I)
 allowed_text_response_actions = frozenset(['suppress', 'claim'])
 
 
+def generate_bucket_id():
+    bucket_id_max = config.get("iris-message-processor", {}).get("number_of_buckets", 100)
+    return randrange(bucket_id_max)
```

Comment:
Document explicitly that bucket_id is a value from 0 to `number_of_bucket-1`, inclusive.